packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -498,6 +500,16 @@ const disableVmHighAvailability = async (xapi: Xapi, vm: Vm) => {
 // - for copies/replications only, added after complete transfer
 //    - `other_config[xo:backup:sr]` = sr.uuid
 //
+// Attributes of created VMs using offline backup mode:
+//
+// - `name_label`: `${original name} - ${job name} - (${safeDateFormat(backup timestamp)})`
+// - `other_config` (these attributes are added at the end of the backup):
+//    - `xo:backup:datetime` = String(Date.now())
+//    - 'xo:backup:job' = job.id
+//    - `xo:backup:schedule` = schedule.id
+//    - 'xo:backup:sr' = sr.$id
+//    - 'xo:backup:vm' = vm.uuid
+//
```

Comment:
Why not merging this with the paragraphs above?

Reply:
Because they don't have the same properties and some properties are not calculated in the same way.